server/portal/apps/workspace/api/views.py

```diff
@@ -25,6 +25,8 @@
 logger = logging.getLogger(__name__)
 METRICS = logging.getLogger('metrics.{}'.format(__name__))
 
+# all the job parameters we want to support for the search
+job_search_parameters = ['name', 'archiveSystemDir', 'appId', 'archiveSystemId']
```

Comment:
Maybe we can keep this within the search logical space since it's only needed there